Description: ugly fix for kernel commit 1ffad83dffd675cd742286ae82dca7d746cb0da8, where an include <linux/if.h> has been added in ip_tables.h
 header file.
 this is conflicting with <net/if.h>, and since we use the latter, I'm adding a guard to avoid the <linux/if.h> inclusion

Author: Gianfranco Costamagna <locutusofborg@debian.org>

--- a/src/firewall-iptables.c
+++ b/src/firewall-iptables.c
@@ -25,6 +25,7 @@
 
 #include <errno.h>
 
+#define _LINUX_IF_H
 #include <xtables.h>
 #include <linux/netfilter_ipv4/ip_tables.h>
 
--- a/src/iptables.c
+++ b/src/iptables.c
@@ -30,6 +30,7 @@
 #include <unistd.h>
 #include <sys/errno.h>
 #include <sys/socket.h>
+#define _LINUX_IF_H
 #include <xtables.h>
 #include <inttypes.h>
 
